lib/OpenQA/Setting.pm

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2017-2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package OpenQA::Setting;
+
+use warnings;
```

Comment:
I wonder if there would be any benefit on getting Mojo::Base imported here.

Reply:
Sorry for that I do not understand your meaning.

Reply:
@Amrysliu Was mostly a random thought, Since you're creating a default constructor for this class, and stuff... and `Mojo::Base -base` does that for you